docs/installation/mac.md

```diff
@@ -69,6 +69,11 @@ archive. We recommend installing from archive on Mac.
 
 ### Installing Elasticsearch from the archive
 
+We recommend that you **do not** install Elasticsearch in the app directory.
+Instead, we recommend installing it at the local root level directory. (This
+also ensures that we don't accidentally commit Elasticsearch code to the
+project's repository!)
+
```

Comment:
A small detail, what's root here? because `cd /` is a read only folder in macOS. My suggestion is to change this to "your home directory, for example `cd $HOME`" or something like that

Reply:
That's a great point, thanks for bringing that up! Latest commit takes your suggestion into account 👍